zebra-network/src/constants.rs

```diff
@@ -95,6 +95,13 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// OS-specific error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_ERROR: &str = if cfg!(unix) {
+    "already in use"
+} else {
+    "access a socket in a way forbidden by its access permissions"
```

Comment:
We might need to turn this into a regex so we can match both "one usage" and "access a socket in a way forbidden by its access permissions", but I want to check the exact error messages for both cases.

Reply:
Windows error from:
https://github.com/ZcashFoundation/zebra/pull/1535/checks?check_run_id=1788192685#step:7:1553

(Note that the other Windows conflict tests passed, so we might be seeing different errors in different tests.)

Full panic message:
```
Opening metrics endpoint listener 127.0.0.1:49817 failed: Hyper(hyper::Error(Listen, Os { code: 10013, kind: PermissionDenied, message: "An attempt was made to access a socket in a way forbidden by its access permissions." })). Hint: Check if another zebrad or zcashd process is running. Try changing the metrics endpoint_addr in the Zebra config.
```

Reply:
The other panic message we see is:
> Opening tracing endpoint listener 127.0.0.1:50715 failed: hyper::Error(Listen, Os { code: 10048, kind: AddrInUse, message: "Only one usage of each socket address (protocol/network address/port) is normally permitted." }). Hint: Check if another zebrad or zcashd process is running. Try changing the tracing endpoint_addr in the Zebra config.

So we'll need a regex.

Reply:
Ok, that should be done now.